package.json

```diff
@@ -80,12 +80,12 @@
     "popper.js": "^1.12.6"
   },
   "devDependencies": {
+    "@babel/cli": "7.0.0-beta.5",
```

Comment:
can update to beta.31 now 😛 https://github.com/babel/babel/releases/tag/v7.0.0-beta.31

It's only 5 -> 31 because babylon was at 30